pom.xml

```diff
@@ -416,7 +416,7 @@
                 <dependencies>
                     <dependency>
                         <groupId>com.oracle</groupId>
-                        <artifactId>ojdbc7</artifactId>
+                        <artifactId>ojdbc8</artifactId>
                         <version>${oracle.version}</version>
```

Comment:
The oracle.version is still set as '12.1.0.2', any reason not to download/set/use the 19.3 version of the jar?

Reply:
Thanks for catching that, I missed it at the first pass.

Reply:
Cool, ty.